code/fireball/fireballs.h

```diff
@@ -137,9 +157,18 @@ int fireball_ship_explosion_type(ship_info *sip);
 // returns the index of the fireball bitmap for this asteroid. -1 if there is none.
 int fireball_asteroid_explosion_type(asteroid_info *aip);
 
+// Gamma39er cinematic wormhole function
+float cutscene_wormhole(float t);
```

Comment:
This function is used exclusively within fireballs.cpp.
And if I judge it right, no other file has any business, realistically, using this function.
As such, the function does not need to / should not be listed in the .h file.
Additionally, it should be a `static float cutscene_wormhole(float t)` to ensure that only fireballs.cpp can use it